conda_build/post.py

```diff
@@ -320,7 +320,7 @@ def mk_relative(m, f):
     if not is_obj(path):
         return
 
-    if sys.platform.startswith('linux'):
+    if sys.platform.startswith('linux') or sys.platform.startswith('openbsd'):
```

Comment:
I understand you don't have other BSDs to test, but given that your changes are so simple, I'd assume they would work on them too.

Well, if `endswith("bsd")` doesn't work, then `bsd in sys.platform` most surely would :-)